backend/src/utils/notebookUtils.ts

```diff
@@ -196,17 +196,17 @@ export const assembleNotebook = async (
   const tolerations: Toleration[] = [];
 
   const affinity: NotebookAffinity = {};
-  if (acceleratorProfile.count > 0 && acceleratorProfile.acceleratorProfile) {
+  if (data.acceleratorProfile?.count > 0 && data.acceleratorProfile.acceleratorProfile) {
```

Comment:
Is the check of `data.acceleratorProfile.acceleratorProfile` still needed? If there is `acceleratorProfile?.count`, there will always be `acceleratorProfile.acceleratorProfile`, right?